ansible/roles/nginx/defaults/main.yml

```diff
@@ -8,13 +8,19 @@ nginx_docker_container: nginx
 nginx_workdir: "/docker/nginx"
 nginx_virtualhosts_workdir: "{{ nginx_workdir }}/conf"
 
-virtualhost:
-  fqdn: example.com
+instances:
+  - virtualhost:
+    fqdn: example.com
+    opensearch_dashboards_endpoint: "https://{{ hostvars[(groups['opensearch_dashboards'][0])].ansible_default_ipv4.address }}:5601"
```

Comment:
Is this the only way to define the endpoint? The problem with this approach is that there's going to be a lot of cut&past of the same string, which is long and prone to error. Can we configure it with another variable, so it's only defined once? (once for the private dashboard and once for the public dashboard).